modules/rosa-hcp-sts-creating-a-cluster-ext-auth-provider-cli.adoc

```diff
@@ -0,0 +1,150 @@
+// Module included in the following assemblies:
+//
+// * rosa_hcp/rosa-hcp-sts-creating-a-cluster-ext-auth.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-hcp-sts-creating-a-cluster-external-auth-provider-cli_{context}"]
+= Creating an external authentication provider
+:source-highlighter: pygments
+:pygments-style: emacs
+:icons: font
+ 
+After you have created a {hcp-title} cluster with external authentication providers, you must create a provider using the ROSA CLI.
+
+[NOTE]
+====
+Similar to the `rosa create|delete|list idp[s]` command in the ROSA CLI, you cannot edit an existing identity provider that you created using `rosa create external-auth-provider`. Instead, you must delete the external authentication provider and create a new one.
+====
+
+The following table shows the possible CLI flags you can use when creating your external authentication provider:
+
+[cols="1,1", options="header"]
+|===
+
+|CLI Flag
+|Description
+
+|`--cluster` 
+|The name or the ID of your cluster
+
+|`--name`
+|A name that is used to refer to the external authentication provider.
+
+|`--console-client-secret`
+|This string is the client secret that is used to associate your account with the application. If you do not include the client secret, this command uses a public OIDC OAuthClient.
+
+|`--issuer-audiences` 
+|This is a comma-separated list of token audiences.
+
+|`--issuer-url`
+|The URL of the token issuer.
+
+|`--claim-mapping-username-claim` 
+|The name of the claim that should be used to construct user names for the cluster identity.
+
+|`--claim-mapping-groups-claim`
+|The name of the claim that should be used to construct group names for the cluster identity.
+
+|===
+
+.Procedure
+
+. Use one of the following commands to create your external authentication provider:
```

Comment:
This is a single step procedure, though like the other one you could probably just move this info into the intro and leave the two options as steps. If you want to keep it here, change the list item type:

```suggestion
* Use one of the following commands to create your external authentication provider:
```